framework/src/controller/bus.ts

```diff
@@ -112,24 +116,28 @@ export class Bus extends EventEmitter2 {
 			},
 		);
 
-		this.rpcServer.expose('invoke', (action, cb: NodeCallback) => {
+		this._rpcServer.expose('invoke', (action, cb: NodeCallback) => {
 			this.invoke(action)
-				.then(data => cb(null, data))
-				.catch(error => cb(error));
+				.then(data => {
+					cb(null, data);
+				})
+				.catch(error => {
+					cb(error);
+				});
 		});
 
-		this.rpcServer.expose('invokePublic', (action, cb: NodeCallback) => {
+		this._rpcServer.expose('invokePublic', (action, cb: NodeCallback) => {
```

Comment:
This can be removed as no longer used

Reply:
Not sure if is used or not, but it was there and have the test coverage as well. Maybe we can remove it afterward and clean all references of `isPublic`

Reply:
Please create an issue for this

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/5509